src/workerd/jsg/modules.h

```diff
@@ -238,6 +239,13 @@ class ModuleRegistryImpl final: public ModuleRegistry {
     entries.insert(Entry(specifier, Type::BUNDLE, kj::fwd<ModuleInfo>(info)));
   }
 
+  void addBuiltinBundle(Bundle::Reader bundle) {
+    for (auto module: bundle.getModules()) {
+      addBuiltinModule(module.getName(), module.getSrc().asChars(),
```

Comment:
Given that capnp is interpreting these as utf8 but the external string approach we use is either strictly one-byte or two-byte representations, we could end up with problems here if the source files contain any non-ascii UTF8 sequences. We don't have to solve for that problem immediately but it would be safer to bundle these as UTF-16 encoded bytes (unless we're certain that the file only contains ascii bytes)